src/components/MakePayment.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState, useEffect } from "react";
+import Button from "./Button";
+import "./CalcPayment.css";
+
+const MakePayment = ({ rates, paymentData, setPaymentData }) => {
+  const [selectedCurrency, setSelectedCurrency] = useState("GBP");
+  const [amount, setAmount] = useState("0.00");
+  const [description, setDescription] = useState("");
+
+  const selectCurrency = (event) => {
+    const currency = event.target.value;
+    setSelectedCurrency(currency);
+  };
+
+  const amountInput = (event) => {
```

Comment:
How about `handleAmountChange` here. This would make better sense as this is a function that "does something"

Reply:
Thank you. I have change all the onChange function names.